src/main/webapp/script.js

```diff
@@ -0,0 +1,33 @@
+loadLectureList();
```

Comment:
Does this work? Can you call the function before it's defined?

Reply:
Yeah it works by hoisting, https://scotch.io/tutorials/understanding-hoisting-in-javascript#:~:text=Hoisting%20is%20a%20JavaScript%20mechanism,scope%20is%20global%20or%20local.

Reply:
Didn't know that, cool!